src/BackupDestination/BackupDestination.php

```diff
@@ -113,6 +115,17 @@ public function connectionError(): Exception
         return $this->connectionError;
     }
 
+    public function extraOptions(): ?array
+    {
+        $extraConfig = config('filesystems.disks.'.$this->diskName().'.backup_extra_options') ?? [];
+
+        if (! is_array($extraConfig) || (count($extraConfig) < 1)) {
```

Comment:
It seems like this check isn't necessary anymore, because you already default it to an empty array.